.gflows/libs/steps.lib.yml

```diff
@@ -20,6 +20,16 @@ name: Checkout repository
 uses: actions/checkout@v2
 #@ end 
 ---
+#@ def _generate_docker_meta():
+name: Docker meta
+id: meta
+uses: docker/metadata-action@v3
+with:
+  images: name/app
+  labels: |
+    org.opencontainers.image.version=candidate-${{ needs.version.outputs.app_version }}
```

Comment:
If I remember correct, there should a function to generate  this label "candidate-xxx": [candidate_image](https://github.com/CoverGo/build-publish-net-workflow/blob/0cc1f79a324424f8bd550073dbb550bfed61f0ba/.gflows/libs/tagging.lib.yml#L21)

Reply:
@andreyleskov  Done